lib/core/README.md

```diff
@@ -3,5 +3,6 @@
 The modules found in `core/` should be modules that are specific to the domain logic of axios. These modules would most likely not make sense to be consumed outside of the axios module, as their logic is too specific. Some examples of core modules are:
 
 - Dispatching requests
+  - requests send at `adapters/` (see lib/adapters/README.md). 
```

Comment:
Can you change this to read:

> Requests sent via `adapters/` (see lib/adapters/README.md)

Reply:
I fixed :) https://github.com/axios/axios/pull/3772/commits/9e5245668a7a1074d88d57c3828c3ad7eb057b74